gf-webapp-admin/src/component/rooms/Rooms.tsx

```diff
@@ -0,0 +1,29 @@
+import React, {useEffect, useState} from "react";
+import {IRoom} from "../../model/room";
+import {RoomsApi} from "../../../autogenerated/gf-server-api/apis/RoomsApi";
+import styles from "./rooms.module.scss";
+import {Room} from "./room/Room";
+
+export const Rooms = () => {
+  const roomsApi = new RoomsApi;
+  const [rooms, setRooms] = useState<IRoom[]>([]);
+
+  const getRooms = async () => {
+    const res = await roomsApi.apiRoomsGet();
```

Comment:
Network requests isn't UI layer. Please, move  roomsApi execution to another layer (service layer)

Reply:
Done